engine/src/main/java/com/odysseusinc/arachne/executionengine/api/v1/AnalysisController.java

```diff
@@ -98,9 +98,11 @@ public AnalysisController(AnalysisService analysisService, CallbackService callb
             consumes = MediaType.MULTIPART_FORM_DATA_VALUE,
             produces = MediaType.APPLICATION_JSON_VALUE
     )
+
     public AnalysisRequestStatusDTO analyze(
             @RequestPart("analysisRequest") @Valid AnalysisRequestDTO analysisRequest,
             @RequestPart("file") List<MultipartFile> files,
+            @RequestPart("container") String container,
```

Comment:
Looks unnecessary

Reply:
This is considered since it was explicitly stated in the ticket: https://odysseusdataservices.atlassian.net/browse/AD20-16 probably for future work.